src/4.3_to_5.0/_addDefaultMeasureIfNoneIsExplicitlyExpressed.ts

```diff
@@ -0,0 +1,51 @@
+import { Cube } from "@activeviam/data-model-5.0";
+import {
+  addMeasure,
+  getMeasures,
+  isMdxDrillthrough,
+  MdxDrillthrough,
+  MdxSelect,
+} from "@activeviam/mdx-5.0";
+
+export function _addDefaultMeasureIfNoneIsExplicitlyExpressed(
+  mdx: MdxDrillthrough,
+  { cube }: { cube: Cube },
+): MdxDrillthrough;
+export function _addDefaultMeasureIfNoneIsExplicitlyExpressed(
+  mdx: MdxSelect,
+  { cube }: { cube: Cube },
+): MdxSelect;
+export function _addDefaultMeasureIfNoneIsExplicitlyExpressed(
+  mdx: MdxSelect | MdxDrillthrough,
+  { cube }: { cube: Cube },
+): MdxSelect | MdxDrillthrough;
+
+/**
+ * When there is any hierarchy expressed on axes, but no measure explicitly expressed, the default measure is:
+ * - displayed in Atoti UI 4.3
+ * - not displayed in Atoti 5.0
+ * This function explicitly adds it to the MDX so that it is isofunctional for the user.
+ */
+export function _addDefaultMeasureIfNoneIsExplicitlyExpressed(
```

Comment:
Given that the default measure is only displayed when there is a hierarchy on columns with no explicit measures, it might be good to modify this function so that the measure is only explicitly added to the query in that scenario. Right now, it adds the measure if there is a hierarchy expressed in either the `rows` or `columns` axis, which is technically not isofunctional to what AUI-4 does.

Reply:
See the behavior of AUI-4 below: 

https://github.com/user-attachments/assets/16253bcc-faff-4adc-b478-f1c05e15f73d